src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -686,25 +683,83 @@ const formConfig = {
+            return ((__BUILDTYPE__ === 'production') && _.get(`disabilities.${index}.view:selectableEvidenceTypes.view:privateMedicalRecords`, formData));
+          },
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:title': disabilityNameTitle,
+                'ui:description': privateRecordsChoice,
+                'view:uploadPrivateRecords': {
+                  'ui:title': 'Do you want to upload your private medical records?',
+                  'ui:widget': 'radio',
+                  'ui:options': {
+                    labels: {
+                      yes: 'Yes',
+                      no: 'No, my doctor has my medical records'
+                    }
+                  }
+                },
+                'view:privateRecords4142Notice': {
+                  'ui:description': download4142Notice,
+                  'ui:options': {
+                    expandUnder: 'view:uploadPrivateRecords',
+                    expandUnderCondition: 'no'
+                  }
+                },
+                'view:privateRecordsChoiceHelp': {
+                  'ui:description': privateRecordsChoiceHelp
+                }
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  required: ['view:uploadPrivateRecords'],
+                  properties: {
+                    'view:uploadPrivateRecords': {
+                      type: 'string',
+                      'enum': ['yes', 'no']
+                    },
+                    'view:privateRecords4142Notice': {
+                      type: 'object',
+                      'ui:collapsed': true,
+                      properties: {}
+                    },
+                    'view:privateRecordsChoiceHelp': {
+                      type: 'object',
+                      properties: {}
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
         privateMedicalRecordRelease: {
           title: 'Private Medical Records Release',
           path: 'supporting-evidence/:index/private-medical-records-release',
           showPagePerItem: true,
           itemFilter: item => _.get('view:selected', item),
           arrayPath: 'disabilities',
           depends: (formData, index) => {
-            const hasRecords = _.get(
-              `disabilities.${
-                index
-              }.view:selectableEvidenceTypes.view:privateMedicalRecords`,
-              formData,
-            );
-            const requestsRecords =
-              _.get(
-                `disabilities.${index}.view:uploadPrivateRecords`,
-                formData,
-              ) === 'no';
-            return hasRecords && requestsRecords;
+            const isNotProd = __BUILDTYPE__ !== 'production';
```

Comment:
You're checking the Prodtype in 3 locations, and setting a variable here only. Would it be beneficial to set this variable in the scope of the file and then reuse within config as needed?